api4/group.go

```diff
@@ -297,6 +297,17 @@ func linkGroupSyncable(c *Context, w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	group, groupErr := c.App.GetGroup(c.Params.GroupId, nil)
+	if groupErr != nil {
+		c.Err = groupErr
+		return
+	}
+
+	if group.Source == model.GroupSourceCustom {
```

Comment:
We want to reject everything except Ldap rather than only rejecting custom.

```suggestion
	if group.Source != model.GroupSourceLdap {
```